float32[] action
---
bool status